src/nvim/os/fs.c

```diff
@@ -85,10 +85,16 @@ int os_dirname(char_u *buf, size_t len)
   FUNC_ATTR_NONNULL_ALL
 {
   int error_number;
+#ifdef WIN32
+  size_t buf_len = len;
+#endif
   if ((error_number = uv_cwd((char *)buf, &len)) != kLibuvSuccess) {
```

Comment:
if we use `len` here, that avoids a `#ifdef`. Then `path_to_long` below, can use `len`. This is better anyways, since it's bad practice to re-assign argument values.